src/createManager.js

```diff
@@ -72,7 +72,7 @@ const protoManager = {
 
   openMenu(openOptions) {
     if (this.isOpen) return;
-    openOptions = openOptions || {};
+    openOptions = openOptions || { focusMenu: true };
     this.isOpen = true;
```

Comment:
What would be the best solution for toggleMenu? Pass the openOptions and closeOptions as separate variables? Like so:

  toggleMenu(openOptions, closeOptions) {
    if (this.isOpen) {
      this.closeMenu(closeOptions);
    // etc.
  }

Or would you prefer wrapping this in a parameter object here too?

I don't think this was implemented in the previous version, but it might be good to add this. If we add this, I will also add documentation to the README.md for toggleMenu in line with the docs for focusMenu and closeMenu.